endpoints/bookstore-grpc/Dockerfile

```diff
@@ -5,11 +5,11 @@ FROM gcr.io/google_appengine/python
 
 # Create a virtualenv for dependencies. This isolates these packages from
 # system-level packages.
-RUN virtualenv /env
+RUN virtualenv -p python3.5 /env
```

Comment:
use `python3.6` if you're gonna use python 3.

Reply:
`python3.6` it is then.